src/SimpleWMSBody.cpp

```diff
@@ -0,0 +1,566 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2019 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "SimpleWMSBody.hpp"
+
+#include "../../../src/cs-core/GraphicsEngine.hpp"
+#include "../../../src/cs-graphics/TextureLoader.hpp"
+#include "../../../src/cs-utils/FrameTimings.hpp"
+#include "../../../src/cs-utils/filesystem.hpp"
+#include "../../../src/cs-utils/utils.hpp"
+
+#define STB_IMAGE_IMPLEMENTATION
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include <stb_image.h>
+#include <stb_image_write.h>
```

Comment:
Do we really need stb_image here? Can't we use the functionality from https://github.com/cosmoscout/cosmoscout-vr/blob/develop/src/cs-graphics/TextureLoader.hpp?

Reply:
You are right, it is already been used, I just forgot to remove the stb_image includes. However, it is still being used in WebMapTextureLoader.cpp. Do you have any idea how to use cs::graphics::TextureLoader with std::future instead?